### PR TITLE
css review

### PR DESCRIPTION
**Pull request by Polkas**
linked to https://github.com/insightsengineering/coredev-tasks/issues/225

- remove inline css rule
- DRY
- style at the head - tags$head(includeCSS())

### CODE REVIEW

File: R/FilterState.R [hunk 1]
```diff
@@ -2156,19 +2156,19 @@ DateFilterState <- R6::R6Class( # nolint
       fluidRow(
         div(
           actionButton(
+            class = "action_button_filter_l",
```
Comment (gogonzo) [inline R/FilterState.R:2159]: maybe `.start_date_rest_button`?

File: R/FilterState.R [hunk 2]
```diff
@@ -2156,19 +2156,19 @@ DateFilterState <- R6::R6Class( # nolint
...
             inputId = ns("start_date_reset"),
             label = NULL,
-            icon = icon("fas fa-undo"),
-            style = "float: left; padding: 0; padding-top: 4px; padding-bottom: 5px; width: 10%;"
+            icon = icon("fas fa-undo")
           ),
           actionButton(
+            class = "action_button_filter_r",
```
Comment (gogonzo) [inline R/FilterState.R:2165]: maybe `.end_date_reset_button`?

File: R/FilterState.R [hunk 3]
```diff
@@ -2156,19 +2156,19 @@ DateFilterState <- R6::R6Class( # nolint
       fluidRow(
         div(
           actionButton(
+            class = "start_date_rest_button",
```
Comment (nikolas-burkoff) [inline R/FilterState.R:2159]: should start and end data have `reset` not `rest`

**Review by nikolas-burkoff — APPROVED** [review]
Just one minor comment if you wish to address it

File: R/FilterState.R (hunk 3 above)
Comment (Polkas) [inline R/FilterState.R:2159]:
```suggestion
            class = "start_date_reset_button",
```

File: R/FilterState.R [hunk 4]
```diff
@@ -2156,19 +2156,19 @@ DateFilterState <- R6::R6Class( # nolint
...
             inputId = ns("start_date_reset"),
             label = NULL,
-            icon = icon("fas fa-undo"),
-            style = "float: left; padding: 0; padding-top: 4px; padding-bottom: 5px; width: 10%;"
+            icon = icon("fas fa-undo")
           ),
           actionButton(
+            class = "end_date_rest_button",
```
Comment (Polkas) [inline R/FilterState.R:2165]:
```suggestion
            class = "end_date_reset_button",
```

File: R/FilterState.R [hunk 5]
```diff
@@ -2473,21 +2473,20 @@ DatetimeFilterState <- R6::R6Class( # nolint
       fluidRow(
         div(
           actionButton(
+            class = "start_date_rest_button",
```
Comment (Polkas) [inline R/FilterState.R:2476]:
```suggestion
            class = "start_date_reset_button",
```

File: R/FilterState.R [hunk 6]
```diff
@@ -2473,21 +2473,20 @@ DatetimeFilterState <- R6::R6Class( # nolint
...
             inputId = ns("start_date_reset"),
             label = NULL,
-            icon = icon("fas fa-undo"),
-            style = "float: left; padding: 0; padding-top: 4px; padding-bottom: 5px; width: 10%;"
+            icon = icon("fas fa-undo")
           ),
           actionButton(
+            class = "end_date_rest_button",
```
Comment (Polkas) [inline R/FilterState.R:2482]:
```suggestion
            class = "end_date_reset_button",
```

File: inst/css/filter-panel.css [hunk 7]
```diff
@@ -1,3 +1,48 @@
...
+
+.m-auto {
+  margin: auto;
+}
+
+.mb-4 {
+  margin-bottom: 1rem;
+}
+
+.start_date_rest_button {
```
Comment (Polkas) [inline inst/css/filter-panel.css:30]:
```suggestion
.start_date_reset_button {
```

File: inst/css/filter-panel.css [hunk 8]
```diff
@@ -1,3 +1,48 @@
...
+  float: left;
+  padding: 0;
+  padding-top: 4px;
+  padding-bottom: 5px;
+  width: 10%;
+}
+
+.end_date_rest_button {
```
Comment (Polkas) [inline inst/css/filter-panel.css:38]:
```suggestion
.end_date_reset_button {
```

File: R/FilterState.R (hunk 3 above)
Comment (Polkas) [reply R/FilterState.R]: thanks

**Pull request merged**